src/generator/index.tsx

```diff
@@ -143,6 +143,8 @@ export default async function renderMessages({ messages, channel, callbacks, ...
           <script
             type="module"
             src={`https://cdn.jsdelivr.net/npm/@derockdev/discord-components-core@${discordComponentsVersion}/dist/derockdev-discord-components-core/derockdev-discord-components-core.esm.js`}
+            // low priority if ssr is enabled
+            defer={options.ssr}
```

Comment:
This won't be used if `ssr` is enabled because we only include the script if `ssr` is _not_ enabled (see above `{!options.ssr && (`)

Besides, I don't think this is a good option. If the transcript is already hydrated, then there is no reason to load the script at all. Defer will still load the script from the CDN, it'll just be done in the background, while the HTML is parsing.

Reply:
i did not mean to commit this, my bad. I was testing things out